### ISSUE TITLE
make configtest results in error

### ISSUE THREAD

**Issue by pwilms** [ysugimoto/aws-lambda-image#6]
cp config.json.sample config.json && make configtest
make configtest

```
==========================================
  AWS-Lambda-Image Configuration Checker
==========================================

Checking configuration format...  Error!
Unexpected token {

/home/ubuntu/workspace/aws-lambda-image/bin/configtest:42
var bucket = ( config.bucket ) ? config.bucket : "";
                     ^
TypeError: Cannot read property 'bucket' of undefined
```

**Comment by pwilms** [comment]
There is a pull request  Missing comma it will fix it

**Comment by ysugimoto** [comment]
Fixed at #5.  thanks
